File: 2.0/java/numbers/findNumbersTollfree.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.numbers.model.Number;
import com.callfire.api.client.api.numbers.model.request.FindTollfreeNumbersRequest;

import java.util.List;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        FindTollfreeNumbersRequest request = FindTollfreeNumbersRequest.create()
            .pattern("87*")
            .limit(10L)
            .fields("number")
            .build();
        List<Number> numbers = client.numbersApi().findNumbersTollfree(request);
    }
}

File: 2.0/csharp/numbers/findNumbersTollfree.cs
using System.Collections.Generic;
using CallfireApiClient;
using CallfireApiClient.Api.Numbers.Model;
using CallfireApiClient.Api.Numbers.Model.Request;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var request = new FindTollfreeNumbersRequest
        {
            Pattern = "87*",
            Limit = 10,
            Fields = "number"
        };
        IList<Number> numbers = client.NumbersApi.FindNumbersTollfree(request);
    }
}
